tests/pos/i3638.scala

```diff
@@ -0,0 +1,12 @@
+package p {
+  package a {
+    class JavaInteraction(arr: Array[Char])
```

Comment:
It looks like this test was minimized from https://github.com/scala/scala/blob/2.13.x/test/files/jvm/protectedacc.scala, it might be worthwhile to port over the whole test (I think it's a run test but it's in /jvm and not /run so it might need to be run in a special way?).

Reply:
I think it can't run on JS because it uses Java classes, that's why it's in jvm. I added the full test to run.